frontend/app/.server/environment/client.ts

```diff
@@ -1,22 +1,23 @@
-import { z } from 'zod';
+import * as v from 'valibot';
+
+import { stringToBooleanSchema } from '../validation/string-to-boolean-schema';
 
 import { buildinfo, defaults as buildinfoDefaults } from '~/.server/environment/buildinfo';
-import * as ValidationUtils from '~/utils/validation-utils';
+import { boolToString } from '~/utils/boolean-utils';
 
-export type Client = Readonly<z.infer<typeof client>>;
+export type Client = Readonly<v.InferOutput<typeof client>>;
 
 export const defaults = {
-  I18NEXT_DEBUG: 'false',
+  I18NEXT_DEBUG: boolToString(false),
```

Comment:
Why? Just use `'false'` here.

Reply:
Humm let me think about it.

Reply:
Because I can easily to a type `'fase'`.

Reply:
In that case we should probably do `AUTH_DEFAULT_PROVIDER: isProduction ? 'azuread'.toLowerCase() : 'local'.toLowerCase()` just in case we accidentally type `AzureAD` or `Local`.

Reply:
😒... I removed it.